### PR TITLE
[ErrorHandler] [ErrorHander] Display exception properties in the HTML error page

### LINKED ISSUE #49613: [ErrorHandler] Display more information about the current exception
### Description

ATM, when an exception is thrown, we display only the message.
But, much more information can be embedded in the exception class.

It's possible to get theses informations via the profiler > logs > errors > context > expand.
It needs 3 click to be able to get them.

I think I would be much more convenient to display all theses information right away on the exception page.

## Before:

image

And with a click in the toolbar > context > expand 

image

## After:

* ⚠️ I made a **very quick and dirty** POC to see if it's doable and easy
* The "exception data" text is a toggle, off by default

image

---

so the question is: Should I open a PR to ship that in SF? (of course, with a better code and look and feel 😂)

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 6.3
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | Fix #49613
| License       | MIT
| Doc PR        |

---

image

---

<details>

<summary>Code to reproduce</summary>

```php

class MyException extends \Exception
{
    public function __construct()
    {
        parent ::__construct('some_message', 0, new MyException2());
    }

    public string $myMessage = 'some_message';
    public string $myCode = 'some_code';
    private string $privateStuff = 'private_stuff';
}

class MyException2 extends \Exception
{
    private string $anotherPrivateStuff = 'another_private_stuff';
}
```

</details>

### CODE REVIEW

File: src/Symfony/Component/ErrorHandler/Resources/views/exception.html.php [hunk 1]
```diff
@@ -23,6 +22,18 @@
...
+        <div class="exception-properties-wrapper">
+            <div class="container">
+                <details>
+                    <summary>Show exception properties</summary>
+                    <div class="">
+                        <?= $this->dumpValue($exception->getProperties()) ?>
+                    </div>
+                </details>
+            </div>
+        </div>
```
Comment (lyrixx) [inline src/Symfony/Component/ErrorHandler/Resources/views/exception.html.php]: If someone could propose a better version, it would be super cool!

**Comment by WebMamba** [comment]
What do you think about adding a new tab? And not be overwelled by too much information on the page load, but we can still access those informations easily.

**Review by yceruto — APPROVED** [review]
LGTM as is, thanks for this improvement!

**Comment by lyrixx** [comment]
> What do you think about adding a new tab?

I think it does not deserves it, and it's really tied to exceptions.

**Review by welcoMattic — APPROVED** [review]
Thank you @lyrixx 👍

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**